packages/ckb-indexer/src/collector.ts

```diff
@@ -12,13 +12,26 @@ import {
   GetCellsResults,
   HexadecimalRange,
   Order,
+  rpcResponse,
   SearchFilter,
 } from "./indexer";
 
 import { CkbIndexer, ScriptType, SearchKey } from "./indexer";
+import axios from "axios";
```

Comment:
axios is too heavy, we can switch to cross-fetch

Reply:
refactor(ckb-indexer): use cross-fetch instand of axios